omero/sysadmins/public.rst

```diff
@@ -138,3 +139,133 @@ authentication. For example a direct link to image will look as follows::
 For more details about how to create an OMERO session see
 :doc:`server-side session </developers/Server/Sessions>` or
 use the :doc:`command line interface </users/cli/sessions>` to create one.
+
+.. _hosting_data_example:
+
+Complete workflow for hosting data for a publication
+----------------------------------------------------
+
+Putting the pieces of this puzzle together, the following describes the steps
+of a complete workflow for using OMERO to host public data associated with a
+publication. It is illustrated using an example publication from the Swedlow
+lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_ with the
+data hosted at
+`<https://omero.lifesci.dundee.ac.uk/pub/schleicher-et-al-2017>`_.
+
+Ansible playbooks can be found describing how the production server in Dundee
+("nightshade") was configured in the
+`prod-playbooks <https://github.com/openmicroscopy/prod-playbooks>`_
+repository on GitHub.
+
+Group setup
+^^^^^^^^^^^
+
+A group-per-publication allows the public user to be selected added (or
```

Comment:
``...selected added (or removed)...`` ?
Maybe ``added (or removed)`` or at least put comma between selected and added ?

Reply:
I think I meant "selectively". Pushing that now.